app/src/navigation/screens/ClaimScreen.tsx

```diff
@@ -130,8 +131,10 @@ const ClaimScreen: React.FC = () => {
   React.useEffect(() => {
     if (userClaim?.type === "EmailCredential" && userClaim.verified) {
       setDisableButton(true);
+      setEmailInput(true);
```

Comment:
set to true wouldn't change anything (default is true right?)

Reply:
I have adjusted it. Thank you :)